src/NitroSharp.NsScript/Primitives/NsColor.cs

```diff
@@ -16,48 +13,68 @@ private NsColor(byte r, byte g, byte b)
         public byte G { get; }
         public byte B { get; }
 
-        public static NsColor Black  => new(0, 0, 0);
-        public static NsColor White => new(255, 255, 255);
-        public static NsColor Red => new(255, 0, 0);
-        public static NsColor Green => new(0, 128, 0);
-        public static NsColor Blue => new(0, 0, 255);
-        public static NsColor Gray => new(128, 128, 128);
+        private static readonly (string, NsColor)[] ColorTable = {
```

Comment:
It's a minor thing, but if you're going to contribute in the future, please try to stick to the same codestyle used throughout the codebase and not randomly switch between different brace styles :meow: